examples/perl_api_example/README.md

```diff
@@ -0,0 +1,30 @@
+
+## Python API Example
+
+### Requirements
+
+* CMake
+* C++ Compiler
+* Perl
+* SWIG
+
+### Running from Command Line
+
+* Run cmake with `BUILD_BINDINGS_PERL=ON`
+* Build target `sourcetraildb`
```

Comment:
I think it's called "bindings_perl", right? It is defined like that in the main CMakeLists.txt file

Reply:
You're right.  This was also corespondingly incorrect in the python_api_example - both corrected.